ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/filtered_tenants.py

```diff
@@ -81,6 +82,10 @@ def filtered_tenants(self: SharedUtils) -> list[dict]:
                         raise AristaAvdError(
                             "WAN configuration requires EVPN to be enabled for VRF 'default'. Got 'address_families: {vrf_default['address_families']}."
                         )
+                # Injecting `wan_vni` on WAN routers if it is missing to make sure that VRF default is always included
```

Comment:
new function for aproved vrf looking at uplink switch vrf and take it if part of this